packages/radio/src/mwc-radio-base.ts

```diff
@@ -155,6 +158,23 @@ export class RadioBase extends FormElement {
     };
   }
 
+  protected rippleHandlers: RippleHandlers = new RippleHandlers(() => {
+    this.shouldRenderRipple = true;
+    return this.ripple;
+  });
+
+  protected renderRipple() {
+    if (this.shouldRenderRipple) {
```

Comment:
Currently this if statement is unsupported in our internal SSR tooling. Can you please refactor this fn to be the same as in checkbox as an ugly ternary? Eventually we should support this pattern, but the tooling is under development.